specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -22033,12 +22025,12 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/workflowConfigurations/{configurationName}": {
-      "get": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/workflowsConnections": {
+      "post": {
         "tags": [
           "WebApps"
         ],
-        "summary": "Get workflow app's configurations connections information by its ID for web site, or a deployment slot.",
+        "summary": "Lists workflow app's configurations connections information for web site, or a deployment slot.",
         "operationId": "WebApps_ListWorkflowsConfiguration",
```

Comment:
Should probably be more descriptive of what's actually happening. 
instead of WebApps_ListWorkflowsConfiguration 
change to "WebApps_ListWorkflowsConnections"

Reply:
Updated!